src/spec/page-objects/billing-page.js

```diff
@@ -7,6 +7,7 @@ class BillingPage {
     this._ccIconAmex = $('.icon-amex');
     this._ccIconVisa = $('.icon-visa');
     this._ccIconMaster = $('.icon-master');
+    this._countrySelect =  element(by.id('country'));
```

Comment:
I prefer `by.id('country')` because is more clear for me, but I verify with `instead of`